Core/include/Acts/TrackFinding/TrackSelector.hpp

```diff
@@ -238,7 +238,6 @@ class TrackSelector {
  private:
   EtaBinnedConfig m_cfg;
   bool m_isUnbinned;
```

Comment:
should we initialize this variable here instead of setting it to `false` in the constructor?

Reply:
good idea.